app/scripts/proactive/view/dom.js

```diff
@@ -785,6 +785,12 @@ define(
             var jobDocumentation = studioApp.models.jobModel.get("Generic Info Documentation");
             var jobGenericInfos = studioApp.models.jobModel.get("Generic Info");
 
+            const submissionMode = {
```

Comment:
Just leave a comment here that explicitly explains this, so that we can remember where this is in the future, in case we want to change how we do this...